CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ set( CMAKE_ALLOW_LOOSE_LOOP_CONSTRUCTS TRUE )
 
 # Define here the needed parameters
 set (OPENRAVE_VERSION_MAJOR 0)
-set (OPENRAVE_VERSION_MINOR 40)
+set (OPENRAVE_VERSION_MINOR 41)
 set (OPENRAVE_VERSION_PATCH 8)

```

Comment:
@bucketzxm if you are to update MINOR version, you can reset PATCH to 0

Reply:
resolved